tomviz/LoadDataReaction.cxx

```diff
@@ -13,6 +13,8 @@
   limitations under the License.
 
 ******************************************************************************/
+#include <vector>
```

Comment:
Try to group includes, this is STL and should be with sstream lower down - alphabetically sorted.